java/ql/src/semmle/code/java/Expr.qll

```diff
@@ -1213,6 +1206,15 @@ class ConditionalExpr extends Expr, @conditionalexpr {
    */
   Expr getFalseExpr() { result.isNthChildOf(this, 2) }
 
+  /**
+   * Gets the expression that is evaluated by the specific branch of this
+   * conditional expression. If `true` that is `getTrueExpr()`, if `false`
+   * it is `getFalseExpr()`.
+   */
+  Expr getBranchExpr(boolean branch) {
+    if branch = true then result = getTrueExpr() else result = getFalseExpr()
```

Comment:
This doesn't bind `branch` so we'll want to replace it with a disjunction of purely positive terms.
```suggestion
    branch = true and result = getTrueExpr() or branch = false and result = getFalseExpr()
```